tests/Bridge/Symfony/Routing/ApiLoaderTest.php

```diff
@@ -113,6 +115,10 @@ private function getApiLoaderWithResourceMetadata(ResourceMetadata $resourceMeta
 
         $kernelProphecy = $this->prophesize(KernelInterface::class);
         $kernelProphecy->locateResource(Argument::any())->willReturn($routingConfig);
+        $containerBuilderProphecy = $this->prophesize(ContainerBuilder::class);
+
+        $kernelProphecy->getContainer()->willReturn($containerBuilderProphecy);
+        $containerBuilderProphecy->has(Argument::any())->willReturn(true);
```

Comment:
As @teohhanhui said, we expect specific values here don't we? So we should check against them :)